src/PKSim.Infrastructure/Reporting/Markdown/Builders/CompoundMarkdownBuilder.cs

```diff
@@ -14,13 +20,49 @@ public CompoundMarkdownBuilder(IMarkdownBuilderRepository markdownBuilderReposit
          _markdownBuilderRepository = markdownBuilderRepository;
       }
 
-      public override void Report(Compound compound, MarkdownTracker tracker)
+      public override void Report(Compound compound, MarkdownTracker tracker, int indentationLevel)
```

Comment:
I am adding this indentationLevel parameter that can be used to specify where the title level (title, subtitle, sub sub title etc).